consensus/subworkflows.nf

```diff
@@ -65,24 +65,57 @@ workflow CygnusAlignedConsensus {
         Minimap2Align(consensus, reference)
         // 3. Rotate by alignment
         RotateByAlignment(Minimap2Align.out)
-
     emit:
         fastq = RotateByAlignment.out
-
 }
+
 workflow CygnusPrimedConsensus {
     take:
         reads_fastq
         primers
-
     main:
         // Its the callers responsibility to make sure primers is a value channel
         Cygnus(reads_fastq)
         RotateBySequence(Cygnus.out, primers)
+    emit:
+        fastq = RotateBySequence.out
+}
 
+workflow Cygnus2Consensus {
+    take:
+        reads_fastq
+    main:
+        Cygnus2(reads)
```

Comment:
Like mentioned for workflow `Cygnus`, most other Cygnus workflows (including for Cygnus2) include process `RotateBySequence`. Is its removal here intended?

Reply:
see above.